ImmuDB4Net/KVMetadata.cs

```diff
@@ -151,32 +181,62 @@ public byte[] Serialize()
     }
 }
 
+/// <summary>
+/// Represents the deleted attribute
+/// </summary>
 public class DeletedAttribute : MetadataAttribute
 {
+    /// <summary>
+    /// Gets the deleted attribute code
+    /// </summary>
+    /// <returns></returns>
     public override byte Code()
     {
         return KVMetadata.deletedAttrCode;
     }
 
+    /// <summary>
+    /// Serializes the attribute
+    /// </summary>
+    /// <returns></returns>
     public override byte[] Serialize()
     {
         return new byte[] { };
     }
 }
 
+/// <summary>
+/// Represents 'the expires at' attribute
+/// </summary>
 public class ExpiresAtAttribute : MetadataAttribute
 {
+    /// <summary>
+    /// Gets or Sets the expires at value
+    /// </summary>
+    /// <value></value>
     public long ExpiresAt { get; set; }
+    /// <summary>
+    /// Creates an instance of ExpiresAtAttribute
+    /// </summary>
+    /// <param name="expiresAt">The expiration time specified in epoch microseconds</param>
```

Comment:
```suggestion
    /// <param name="expiresAt">The expiration time specified in epoch seconds</param>
```

Reply:
done

Reply:
Actually I think it would be good here to accept the DateTime parameter - WDYT?
Of course documentation would have to remark that it will be truncated to second precision

Reply:
done, added an overloaded constructor